management-core/src/main/java/com/ns/task/service/ProductCoreServiceImpl.java

```diff
@@ -17,18 +17,27 @@
 
 @Service
 public class ProductCoreServiceImpl implements ProductService {
-
-    private final ModelMapper mapper;
-    private final ProductRepository repository;
+    private ModelMapper mapper;
+    private ProductRepository repository;
     private static final Logger logger = LogManager.getLogger();
 
+    public ProductCoreServiceImpl() {
```

Comment:
Do you need the empty constructor?

Reply:
Yes you are right, do you know a tool that give some advices for this unused code? because I just installed checkstyle but it not giving me this kind of information